WcaOnRails/vendor/assets/javascripts/jquery.wca-autocomplete.js

```diff
@@ -73,6 +75,14 @@
             return $div[0].outerHTML;
           },
 
+          regulation: function(regulation) {
+            var $div = $('<div class="wca-autocomplete-regulation"><span class="id"></span>: <span class="text"></span></div>');
+            $div.find(".id").text(regulation.id);
+            // NOTE: the regulation 'text' field may contain links (references to other regulations/guidelines)
```

Comment:
This feels like something people are going to easily forget. How do you feel about renaming this to `html`, or maybe `content_html`? I feel like that would make it harder for people to miss this.